cheat-sheet.js

```diff
@@ -0,0 +1,268 @@
+            let context = this;
+            let args = arguments;
+
+            let later = function () {
+                timeout = null;
+                if (!immediate) func.apply(context, args);
+            };
+
+            let callNow = immediate && !timeout;
+
+            clearTimeout(timeout);
+
+            timeout = setTimeout(later, wait);
+
+            if (callNow) func.apply(context, args);
+        };
+    }
+
+    function addSearchResultItemToFragment(fragment, iconClassName, codepoint) {
+        let parentDiv = document.createElement("div");
+        parentDiv.setAttribute("class", "column");
+
+        let index = iconClassName.indexOf('-');
+        let namespace = iconClassName.substring(0, index);
+        let iconClassNameBeforeDeprecation = iconClassName;
+
+        if (namespace === "nfold") {
+            let rest = iconClassName.substring(index + 1);
+            iconClassNameBeforeDeprecation = `nf-${rest}`;
+
+            let span1 = document.createElement("span");
+            span1.setAttribute("class", "corner-red");
+            parentDiv.appendChild(span1);
+
+            let span2 = document.createElement("span");
+            span2.setAttribute("class", "corner-text");
+            span2.textContent = "removed";
+            parentDiv.appendChild(span2);
+        }
+
+        let innerDiv1 = document.createElement("div");
+        innerDiv1.setAttribute("class", `center ${namespace} ${iconClassName}`);
+        parentDiv.appendChild(innerDiv1);
+
+        let innerDiv2 = document.createElement("div");
+        innerDiv2.setAttribute("class", "class-name");
+        innerDiv2.textContent = iconClassNameBeforeDeprecation;
+        parentDiv.appendChild(innerDiv2);
+
+        let innerDiv3 = document.createElement("div");
+        innerDiv3.setAttribute("class", "codepoint");
+        innerDiv3.setAttribute("title", "Copy Hex Code to Clipboard");
+        innerDiv3.textContent = codepoint;
+        parentDiv.appendChild(innerDiv3);
+
+        fragment.appendChild(parentDiv);
+    }
+
+    function loadMoreSearchResults() {
+        let fragment = document.createDocumentFragment();
+        remainingSearchResults.slice(0, maxSearchResults).forEach(result => {
+            addSearchResultItemToFragment(fragment, result.id, result.code);
+        });
+        elementGlyphCheatSheet.append(fragment);
+
+        remainingSearchResults = remainingSearchResults.slice(maxSearchResults, -1);
+    }
+
+    function searchGlyphs() {
+        let searchTerm = elementGlyphSearch.value;
+
+        if (searchTerm === "") {
+            // MiniSearch don't allow empty searches, so we search for "nf".
+            searchTerm = "nf";
+        }
+
+        // TODO: search suggestions
+
+        remainingSearchResults = miniSearch.search(searchTerm,
+            {
+                prefix: true,
+                combineWith: "AND",
+                boostDocument: ((documentId, term, storedFields) => {
```

Comment:
The downside of the `boost` based sorting is ... that it follows documents text searcher rules.
(`file_code` in front of `account`)

Take for example this...

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/89c02a4e-3e3d-46d5-9060-f22346410aab)

Because `eae9` has `cod` in the name twice it is listed too early (if you expect a sort-by-name list).

Reply:
> The downside of the boost based sorting is ... that it follows documents text searcher rules.

It's not a downside to boost based sorting, but the consequence of setting `prefix: true`